### ISSUE TITLE
Request: Liveblocks

### ISSUE THREAD

**Issue by SebastianZxD** [simple-icons/simple-icons#12098]
### Brand Name

Liveblocks

### Website

https://liveblocks.io/

### Popularity Metric

The Similarweb global rank is: 284,687. See https://www.similarweb.com/website/liveblocks.io for details.

### Official Resources for Icon and Color

![liveblocks](https://github.com/user-attachments/assets/8a81b9ba-983c-418d-bbc0-77ad73bd2266)

**Comment by uncenter** [comment]
As per https://liveblocks.io/terms:

> Trademarks: The Liveblocks name and logos are trademarks and service marks of Liveblocks (collectively the “Liveblocks Trademarks”). Other Liveblocks, product, and service names and logos used and displayed via the Service may be trademarks or service marks of their respective owners who may or may not endorse or be affiliated with or connected to Liveblocks. Nothing in these Terms of Service or the Service should be construed as granting, by implication, estoppel, or otherwise, any license or right to use any of Liveblocks Trademarks displayed on the Service, without our prior written permission in each instance. All goodwill generated from the use of Liveblocks Trademarks will inure to our exclusive benefit.

We should seek permission to include their logo first. @SebastianZxD would you be willing to reach out to the company? We have some instructions here: https://github.com/simple-icons/simple-icons/blob/develop/CONTRIBUTING.md#requesting-permission.

**Comment by SebastianZxD** [comment]
Yes, i'll try reaching out to them in order to get permission to add their logo.

**Comment by PeterShaggyNoble** [comment]
See #13453
